lib/lazy-videos/lazy_tiktok.rb

```diff
@@ -11,16 +11,21 @@ def to_html
          oembed_data.embed_product_id
       thumbnail_url = oembed_data.thumbnail_url
       escaped_title = ERB::Util.html_escape(oembed_data.title)
+      video_height = oembed_data.thumbnail_height < 1024 ? 998 : oembed_data.thumbnail_height
+      height = (323.0 / 576) * video_height
 
       <<~HTML
         <div class="tiktok-onebox lazy-video-container"
           data-video-id="#{oembed_data.embed_product_id}"
           data-video-title="#{escaped_title}"
-          data-provider-name="tiktok">
+          data-provider-name="tiktok"
+          data-height=#{height}
+          style="height: #{height + 7}px">
```

Comment:
Why `+7`? To account for borders?

Reply:
yes, exactly
top: 1px border from the iframe + 3px border.
bottom: 4px border

Reply:
Can we find a way to avoid it? Asking because in the future (or in a theme) someone can edit the border in the stylesheet and won't know that the template relies on the border's size. 

Is `border-box` helpful at all?

Reply:
I agree. Not a pretty solution. `border-box` might help. I'll try to find a better workaround